window.lua

```diff
@@ -0,0 +1,28 @@
+local String = require 'stringutil'
+
+local Window = {}
+
+local WINDOW_FILE = 'window.data'
+
+Window.initialize = function()
+  local fileInfo = love.filesystem.getInfo(WINDOW_FILE)
+  if fileInfo then
+    local contents = love.filesystem.read(WINDOW_FILE)
+    local windowArr = String.split(contents, ',')
```

Comment:
Probably won't need to trim given no-one in their right mind would modify this. But you could if someone is outside of their mind.

Reply:
It's coded into my brain now.